definitions/features/installer.rb

```diff
@@ -105,13 +105,19 @@ def installer_arguments
     installer_args = ''
 
     if feature(:foreman_proxy) &&
-       feature(:foreman_proxy).with_content?
+       feature(:foreman_proxy).with_content? &&
+       !feature(:katello_install)
       installer_args += ' --disable-system-checks'
```

Comment:
I think this option with whole if block can be removed now?

Reply:
This was dropped for Foreman 3.4+.

Reply:
And for vanilla Foreman it never existed in the first place.